### ISSUE TITLE
Duplicate entry in vertex shader

### ISSUE THREAD

**Issue by LeXXik** [playcanvas/engine#5511]
The current vertex generation produces a duplicate definition entry. For example "forward" pass becomes:

```glsl
...
#define FORWARD_PASS
#define FORWARD_PASS
...
#
```

This takes place here:

https://github.com/playcanvas/engine/blob/3b221f508a6d0df82674da59047d9d148a4e341a/src/scene/shader-pass.js#L54-L58

**Comment by mvaligursky** [comment]
Yep thanks, I was aware of it when adding this code. It's duplicate in some cases, but not in others. Easy to fix by comparing and not adding if the same.

**Comment by mvaligursky** [comment]
Actually, I'll close it. I thought about it when I added it few months ago and I was ok with it. In many cases it's the same, but it cases no problems.
